src/applications/facility-locator/components/LocationHours.jsx

```diff
@@ -1,27 +1,61 @@
 import { values, every, capitalize } from 'lodash';
 import React, { Component } from 'react';
+import Raven from 'raven-js';
+import moment from 'moment';
 
 /**
  * VA Facility Known Operational Hours
  */
 export default class LocationHours extends Component {
-  colonizeTime(time) {
-    const found = time.match(/(\d?\d)(\d\d)(\w\w)/);
-    return `${found[1]}:${found[2]}${found[3]}`;
+  convertHour(hour, day) {
+    const hours = hour.split('-');
+    const isValid = hours.every(time => moment(time, 'hmmA').isValid());
+
+    if (!isValid) {
```

Comment:
@jbalboni The extra property in  Raven.captureMessage object was an example I found on how to add extra data to the error log. I'm not totally sure if it works.  Here is where I got it from. Is there a way to check if this got recorded in Sentry?

https://github.com/getsentry/sentry-javascript/issues/167